samplers/jaegerremote/sampler_remote_options.go

```diff
@@ -37,6 +40,36 @@ type config struct {
 	logger                  logr.Logger
 }
 
+func getEnvOverrideOptions() []Option {
+	var options []Option
+	if os.Getenv("OTEL_TRACES_SAMPLER") == "jaeger_remote" {
+		args := strings.Split(os.Getenv("OTEL_TRACES_SAMPLER_ARG"), ",")
+		for _, arg := range args {
+			keyValue := strings.Split(arg, "=")
+			if len(keyValue) != 2 {
+				continue
+			}
+			switch keyValue[0] {
+			case "endpoint":
+				options = append(options, WithSamplingServerURL(keyValue[1]))
+			case "pollingIntervalMs":
+				intervalMs, err := strconv.Atoi(keyValue[1])
+				if err != nil {
+					continue
+				}
+				options = append(options, WithSamplingRefreshInterval(time.Duration(intervalMs)))
+			case "initialSamplingRate":
+				samplingRate, err := strconv.ParseFloat(keyValue[1], 64)
+				if err != nil {
+					continue
+				}
```

Comment:
Hi @yurishkuro  , does it work to ignore errors here or should we report it to the user?

Reply:
errors should be logged using the user-provided logger

Reply:
Done